sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -44,33 +48,32 @@ export interface AttestationCertificateManagementBody {
 }
 
 // @public (undocumented)
-export class AttestationClient extends AttestationClientContext {
-    constructor(credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials, instanceUrl: string, options?: AttestationClientOptionalParams);
+export class AttestationClient {
+    constructor(credentials: TokenCredential, instanceUrl: string, options?: AttestationClientOptions);
     // (undocumented)
     attestation: Attestation;
+    // Warning: (ae-forgotten-export) The symbol "GeneratedClient" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    BaseClient(): GeneratedClient;
+    // (undocumented)
+    getAttestationSigners(options?: AttestationClientOperationOptions): Promise<AttestationSigner[]>;
     // (undocumented)
-    metadataConfiguration: MetadataConfiguration;
+    getOpenIdMetadata(options?: AttestationClientOperationOptions): Promise<any>;
+    // (undocumented)
+    instanceUrl: string;
```

Comment:
should that be private?

Reply:
Eventually, probably. For now, it's needed outside the attestationClient class.